package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-demo-payment-service",
-  "version": "3.4.0",
+  "version": "3.4.1",
```

Comment:
For consistency with the other changes, I think this should be a feature increment.